doc/tools/make_rst.py

```diff
@@ -1500,7 +1500,7 @@ def resolve_type(link_type: str) -> str:
         return f":ref:`Array<class_Array>`\\[{resolve_type(klass[: -len('[]')])}\\]"
 
     if klass.startswith("Dictionary["):  # Typed dictionary, split elements to link contained types.
-        parts = klass[len("Dictionary[") : -len("]")].partition(", ")
```

Comment:
the spacing around colons was autoformatted by my IDE. 

is there a styleguide for python that I should follow?

should I reintroduce the spaces?